app.yaml

```diff
@@ -0,0 +1,4 @@
+runtime: custom
+env: flex
```

Comment:
any reason we need flex vs. cloud run or just a job on k8s?

Reply:
It was quick and simple to get going. Just those two lines, and the single app-deploy command, no other config needed.

Looks like cloud run throttles off the CPU when the app is not serving a request, doesn't work well with background tasks, which we need. They have a feature to send your app a request periodically to solve that, so I could rework the code and setup that request to work on cloud run. Might be an option.

K8s will work, but I just need one VM one pod one process for now. I might go that way because deploying to flex takes forever.